cmd/main.go

```diff
@@ -95,20 +101,50 @@ func main() {
 	if k8sConfig.Host != "https://"+net.JoinHostPort(host, port) {
 		leaderElectionNamespace = "default"
 	}
+	leaseDuration := os.Getenv(envLeaseDuration)
+	if leaseDuration == "" {
+		leaseDuration = defaultLeaseDuration.String()
+	}
+	leaseDurationDuration, err := time.ParseDuration(leaseDuration)
+	if err != nil {
+		log.Error(err, "unable to parse lease duration")
+		os.Exit(1)
+	}
```

Comment:
This seems a little overcomplicated for me, I don’t think converting the duration to string and back, or exiting on failure is necessary, what about this:

```
leaseDuration := time.ParseDuration(os.Getenv(envLeaseDuration))
if leaseDuration == "" {
  leaseDuration = defaultLeaseDuration
}
```

… and the same for the others